### DISCUSSION TITLE
Shelly Button zum sofortigen Starten

### DISCUSSION THREAD

**Discussion by charly2blue** [camueller/SmartApplianceEnabler#395] [Allgemein]
Hallo zusammen,

noch läuft meine PV-Anlage nicht, da der WR noch nicht lieferbar ist. Aber der Raspi mit dem SAE läuft schon. ;-)

Nun bin ich gerade dabei die Shellys zu verbinden und Schalter und Zähler zu konfigurieren. Waschmaschine und Geschirrspüler sollen per Anlaufstromerkennung angehalten werden. Die Dame des Hauses möchte das jedoch gerne "overrulen" können, sie ist öfters im HomeOffice und wenn Sie dann Wäsche waschen will, ist es oft mehr als eine Ladung und dann soll es natürlich sofort losgehen.

Idee dazu wäre einfach einen Shelly-Button zu nehmen und dem entsprechenden Shelly-Relais den Startbefehl direkt per HTTP-Befehl zu geben. Aber was sagt der SAE dazu? Hält der die Waschmaschine wieder an? Er erkennt ja einen erneuten Anlaufstrom.

Gruß Christoph

**Comment by camueller** [comment]
Manuelle Steuerung erfolgt über die Ampel: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Status_DE.md

**Reply by charly2blue** [reply]
Danke für deine schnelle Antwort.
Das würde ich auch so machen, aber wenn du meiner Frau erzählst sie möge sich bitte ihr Handy nehmen, die Seite des SAE aufmachen und die entsprechende Ampel anwählen um die Waschmaschine zu starten....
Keine Chance, damit wäre die Einbindung von Waschmaschine, usw. in den SAE definitiv raus. Einen Knopf direkt neben der Waschmaschine drücken würde da auf deutlich mehr Akzeptanz stoßen.

**Comment by camueller** [comment]
Meine Freundin macht das mit dem Handy. Das hat man ja meist sowieso in der Nähe. Und ausserdem geht es ja nur um das manuelle Erzwingen einens sofortigen Starts, das sollte ja nicht der Normalfall sein.

Falls Du das mt dem Shelly-Button machen willst, müsste er dem SAE den selben Befehl schicken, wie der SHM es macht. Siehe https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/REST_DE.md

**Reply by charly2blue** [reply]
Alles klar, das schaue ich mir mal genauer an
